library/src/main/java/com/novoda/simplechromecustomtabs/SimpleChromeCustomTabs.java

```diff
@@ -67,7 +59,6 @@ public WebNavigator withFallback(NavigationFallback navigationFallback) {
      * Provides a {@link IntentCustomizer} to be used to customize the Chrome Custom Tabs by attacking directly to
      * {@link SimpleChromeCustomTabsIntentBuilder}
      *
-     * @param intentCustomizer
```

Comment:
An `IntentCustomizer` provides functionality to style the chrome custom tabs by allowing the user to set the colour, menu items, or making the url be hidden between others